azkaban-web-server/src/main/java/azkaban/imagemgmt/servlets/ImageTypeServlet.java

```diff
@@ -74,17 +88,71 @@ protected void handleGet(
       throws ServletException, IOException {
     try {
       final Map<String, String> templateVariableToValue = new HashMap<>();
-      if (IMAGE_TYPE_WITH_ID_URI_TEMPLATE.match(req.getRequestURI(), templateVariableToValue)) {
-        // TODO: Implementation will be provided in the future PR
+      if (IMAGE_TYPE_NAME_OR_ID_URI_TEMPLATE.match(req.getRequestURI(), templateVariableToValue)) {
+        getImageTypeDTOByIdOrImageTypeName(resp, session, templateVariableToValue);
       } else if (GET_IMAGE_TYPE_URI.equals(req.getRequestURI())) {
-        // TODO: Implementation will be provided in the future PR
+        ImageTypeDTO imageTypeDTO;
+        if (!StringUtils.isEmpty(req.getQueryString()) && req.getQueryString()
+            .contains(IMAGE_TYPE)) {
```

Comment:
Is the `StringUtils.isEmpty()` check needed? Can `req.getQueryString()` return null?

Reply:
No, I don't believe it is. I will remove it. Thanks.